routes/user.signup.js

```diff
@@ -79,6 +79,9 @@ router.post('/', function(req, res) {
             password: password
         });
 
+        //this temp user will be used to check whether the username is already exist or not
+        //this has to be done because nev.createTempUser from email-verification api just handle the account
```

Comment:
yes. this part is for solving the issue when one user signs up with a username and he hasnt verified it, and there is another person who tries to sign up with the same username.